backend/src/main/kotlin/rocks/didit/sefilm/database/entities/ParticipantInfo.kt

```diff
@@ -2,16 +2,18 @@ package rocks.didit.sefilm.database.entities
 
 import org.springframework.data.annotation.Id
 import org.springframework.data.mongodb.core.mapping.Document
+import rocks.didit.sefilm.domain.Participant
 import rocks.didit.sefilm.domain.SEK
 import rocks.didit.sefilm.domain.UserID
 import java.util.*
 
 @Document
 data class ParticipantInfo(
-  @Id
-  val id: UUID = UUID.randomUUID(),
-  val userId: UserID = UserID(),
-  val showingId: UUID = UUID.randomUUID(),
-  val hasPaid: Boolean = false,
-  val amountOwed: SEK = SEK(0))
+        @Id
+        val id: UUID = UUID.randomUUID(),
+        val participant: Participant,
```

Comment:
This should have a default value. Either provide a default Participant of some kind, or make it a nullable type.

The reason being that Jackson (serialization) does not take kindly to classes without a default constructor.